easybuild/easyconfigs/p/p7zip/p7zip-9.38.1-GCC-4.9.2.eb

```diff
@@ -0,0 +1,29 @@
+easyblock = 'MakeCp'
+
+name = 'p7zip'
+version = '9.38.1'
+
+homepage = 'http://p7zip.sourceforge.net/'
+description = """p7zip is a quick port of 7z.exe and 7za.exe (command line version of 7zip) for Unix.
+ 7-Zip is a file archiver with highest compression ratio."""
+
+toolchain = {'name': 'GCC', 'version': '4.9.2'}
+
+source_urls = [SOURCEFORGE_SOURCE]
+sources = ['p7zip_%(version)s_src_all.tar.bz2']
+
+prebuildopts = "cp makefile.linux_amd64  makefile.linux &&"
+buildopts = 'all3 CC="$CC" CXX="$CXX" OPTFLAGS="$CFLAGS"'
+
+files_to_copy = ['bin']
+
+sanity_check_paths = {
+    'files': ['bin/7z', 'bin/7za', 'bin/7zCon.sfx', 'bin/7zr', 'bin/7z.so', 'bin/Codecs/Rar29.so'],
```

Comment:
`ldd` doesn't show that `7z` requires `7z.so` at all, so it goes hunting for `7z.so` itself somehow (otherwise I would've just set `$LD_LIBRARY_PATH`)

so static linking can't help us here :)

note that `7z` also doesn't work unless it's called full path (hence the alias below); otherwise, it complains that `./7z.so` is not found... >_<